adoc/chapters/introduction.adoc

```diff
@@ -3,154 +3,59 @@
-The SYCL standard does not preclude the use of a single compiler flow, but is
-designed to not require it.
-SYCL can also be implemented purely as a library, in which case no special
-compiler support is required at all.
-
-The advantages of this design are two-fold.
-First, it offers better integration with existing tool chains.
-An application that already builds using a chosen compiler can continue to do so
-when SYCL code is added.
-Using the SYCL tools on a source file within a project will both compile for a
-device and let the same source file be compiled using the same host compiler
-that the rest of the project is compiled with.
-Linking and library relationships are unaffected.
-This design simplifies porting of pre-existing applications to SYCL.
-Second, the design allows the optimal compiler to be chosen for each device
-where different vendors may provide optimized tool-chains.
-
-To summarize, SYCL enables computational kernels to be written inside {cpp}
-source files as normal {cpp} code, leading to the concept of "`single-source`"
-programming.
-This means that software developers can develop and use generic algorithms and
-data structures using standard {cpp} template techniques, while still supporting
-multi-platform, multi-device heterogeneous execution.
-Access to the low level APIs of an underlying implementation (such as OpenCL) is
-also supported.
-The specification has been designed to enable implementation across as wide a
-variety of platforms as possible as well as ease of integration with other
-platform-specific technologies, thereby letting both users and implementers
-build on top of SYCL as an open platform for system-wide heterogeneous
-processing innovation.
+// What is SYCL?
+SYCL (pronounced "`sickle`") is a royalty-free, cross-platform API for
+heterogeneous computing in {cpp}.
+
+SYCL enables developers to write standard {cpp} code that executes on a wide
+range of devices, using modern techniques such as inheritance, templates, and
+lambda functions.
+All computational kernels to be executed on a device can be written inside {cpp}
+source files as normal {cpp} code, alongside any code intended to be run on a
+system's host processor.
+This concept, known as "`single-source`" programming, reduces the complexity of
+heterogeneous programming for developers and gives compilers greater
+opportunities to optimize across the host-device boundary.
+
+// How does SYCL relate to C++?
+SYCL is designed to be as close to standard {cpp} as possible, and some
+implementations of SYCL may be able to use a standard {cpp} compiler to target
+CPU devices.
+However, to ensure portability of device code across a wide range of devices,
+SYCL imposes some restrictions on the set of {cpp} features that SYCL
+implementations are required to support within device code.
+These restrictions may not be applicable to all devices and can therefore be
+relieved by specific Khronos extensions or vendor extensions.
```

Comment:
Could *relaxed* work better than *relieved*? *Relieved* makes it sound like using core SYCL without extensions will be particularly distressing.

Reply:
Yes, I agree that's better.  Fixed in f34dabe.